src/main/java/lotto/LottoGame.java

```diff
@@ -0,0 +1,75 @@
+package lotto;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoGame {
+
+    public static final int LOTTO_NUM_START = 1;
+    public static final int LOTTO_NUM_END = 45;
+    public static final int LOTTO_PICK_NUMBERS = 6;
+
+    private final List<Integer> numbers;
+
+    public LottoGame(List<Integer> numbers) {
+        validateNumbers(numbers);
+
+        this.numbers = numbers;
+    }
+
+    public static void validateNumbers(List<Integer> numbers) {
+        if (numbers.size() != LOTTO_PICK_NUMBERS) {
+            throw new IllegalArgumentException("choose " + LOTTO_PICK_NUMBERS + " numbers");
+        }
+
+        if (numbers.stream().anyMatch(n -> !isValidNumber(n))) {
```

Comment:
```suggestion
        if (numbers.stream().anyMatch(n -> isInvalidNumber(n))) {
```

!를 붙이는 부정문은 가독성을 해칠 수 있습니다!
Objects.isNull() Objects.nonNull() 두 메서드를 제공하는 이유에 대해 고민해보시면 어떨까요? 😄